README.adoc

```diff
@@ -311,4 +311,22 @@ Older versions of `CMake`` only honor this option with the Intel compiler on Lin
 This option *breaks* safe pointers, resulting in crashes, therefore is suitable only for debugging.
 When combined with `-DRUNTIME_CHECK=asan`, the pointer breakages are much less frequent.
 
+|`-DBUILD_TESTING=OFF`
+|Excludes project's tests from the build.
```

Comment:
This is a standard CMake feature, see https://cmake.org/cmake/help/latest/module/CTest.html.

Dispatch does not appear to fully support it, though it should. Reported as #172.

Reply:
To be fixed by #177

Reply:
Merged